RELEASING.md

```diff
@@ -0,0 +1,77 @@
+# Making a new release
+
+This guide describes the process of releasing osbuild both [upstream][upstream-git] and into [Fedora][fedora-distgit] and [CentOS Stream][centos-distgit].
+
+> This guide assumes that the number of the new release is stored in an environment variable called `VERSION`.
+
+## Upstream release
+
+Osbuild release is a tagged commit that's merged into the `main` branch as any other contribution.
+
+### Making a release PR
+
+Firstly, a new branch from (up-to-date) main needs to be created:
+
+```
+git checkout main
+git pull
+git checkout -b release-version-$VERSION
+```
+
+Compile all the news from the pull requests associated with the version milestone on GitHub!
+
+Bump versions in osbuild.spec and setup.py.
+
+```
+make release
+make bump-version
+```
+
+At this moment, everything in the checkout is ready for making the release commit:
+
+```
+git add osbuild.spec NEWS.md setup.py
+git commit -s -m $VERSION -m "Release osbuild $VERSION"
+```
+
+As the final step, push the branch into your fork and open a PR against the `main` branch. The PR now needs to be approved and merged.
+
+### After the release PR is merged
+
+Firstly, make sure to update the local checkout of the `main` branch:
+
+```
+git checkout main
+git pull
+```
+
+Tag the release and push the tag:
+
+```
+git tag -s -m 'osbuild $VERSION' v$VERSION HEAD
```

Comment:
Maybe change this to double quotes?  Just in case people follow it blindly and use the `$VERSION` var and end up with that as the literal tag comment.